README.md

```diff
@@ -56,7 +54,11 @@ Run local App Engine dev server:
 Configure your local development parameters by copying the `dev_config/config.yml.dist` file to `dev_config/config.yml`.  Edit `config.yml` as needed.  This file is .gitignore'd.  See comments in the `config.yml.dist` file for more details.
 
 #### MySQL database configuration
-Create a new MySQL for this application.  Configure the MySQL connection in `config.yml`.  Then, run `bin/console doctine:migrations:migrate` to build your local database.
+Create a new MySQL for this application.  Configure the MySQL connection in `config.yml`. 
+
+Then, run `bin/console doctine:migrations:migrate` to build your local database.
```

Comment:
Did you mean to leave this here? Not sure if this was accidental or if it's still a good idea to have a local database configured outside of Docker since we still run the deploy from the host machine.

Reply:
Yes, I left it intentionally since the deploy command runs the tests and needs local db access.